README.md

```diff
@@ -47,6 +47,76 @@ Click **Add user** and fill in the required details, and check the “**Allow AP
 
 For more instructions, check out examples folder.
 
+## A usage example for creating a server
+
+Below is an example configuration on how to create a server using the Terraform provider. Note that currently, only 10 concurrent servers creation is possible per account. For more examples, visit [official Terraform documentation](https://registry.terraform.io/providers/UpCloudLtd/upcloud/latest/docs).
+
+```
+provider "upcloud" {
```

Comment:
```
$ terraform init

Initializing the backend...

Initializing provider plugins...
- Finding latest version of hashicorp/upcloud...

Error: Failed to install provider

Error while installing hashicorp/upcloud: provider registry
registry.terraform.io does not have a provider named
registry.terraform.io/hashicorp/upcloud
```

The beginning of the file is a bit off https://registry.terraform.io/providers/UpCloudLtd/upcloud/latest?pollNotifications=true as we see from here using the provider needs to start like this:
```
terraform {
  required_providers {
    upcloud = {
      source = "UpCloudLtd/upcloud"
      version = "1.0.0"
    }
  }
}

provider "upcloud" {
  # Configuration options
}
```